desktop/core/src/desktop/js/ko/bindings/ko.vue.js

```diff
@@ -16,17 +16,24 @@
 
 import * as ko from 'knockout';
 
+function setProps(el, props) {
+  for (const key in props) {
+    el.setAttribute(key, props[key]);
```

Comment:
Why the attribute? It seems to me like it will set attr="[object Object]" most of the time... Perhaps add a typeof check if needed.

Reply:
Yup... but that calls the wrapper's MutationObserver and in turn refreshes the inlying web component component.